app/assets/stylesheets/darkswarm/_shop-popovers.css.scss

```diff
@@ -25,54 +24,27 @@
 
   .joyride-nub.right {
     top: 38px;
-    border-color: #999 !important;
+    border-color: $grey-800 !important;
     border-top-color: transparent !important;
     border-right-color: transparent !important;
     border-bottom-color: transparent !important;
   }
 
-  .progress {
-    background-color: #13bf85;
-    padding: 0;
-    border: none;
-    color: white;
-    font-size: 0.75rem;
-    font-style: oblique;
-    line-height: 1;
-    height: auto;
-
-    .right {
-      padding: 0.5rem 0.25rem 0 0;
-    }
-
-    .meter {
-      background-color: #0b8c61;
-      padding: 0.5rem 0.25rem;
-      border-right: 1px solid #539f92;
-    }
+  ul, li {
+    list-style: none;
+    margin: 0;
+    font-size: 0.875rem;
```

Comment:
we need to come up with a set of defined font-size vars as we do with branding.css so we pick consistent values. This won't be easy but we got to start agreeing on a convention as we move along.

I did a bit of gripping and so far we have around 72 different values for `font-size` across the app :boom: this can't end well. But this `0.875rem;` seems to be a pretty standard-ish font size.

I suggest we start adding a var for the font-sizes you are using here in https://github.com/openfoodfoundation/openfoodnetwork/blob/master/app/assets/stylesheets/darkswarm/typography.css.scss. It seems the team came up with a similar idea with `.text-small` there. This `0.875rem;`.

Then, after this PR, we can think of how to reuse these vars and not dying in the effort. Thoughts?

Reply:
Agreed! We should figure out `rem` and `em` guidelines. I think `em` is generally good for font-sizes but maybe `rem` for padding and margin..?

Reply:
I agree that it's a mess and it would be good to standardise these things. I would prefer to do that in another PR though because this one has 25 commits already.